tools/localazy/generateLocalazyConfig.py

```diff
@@ -36,10 +36,13 @@ def convertModuleToPath(name):
 # Iterating on the config
 for entry in config["modules"]:
     # Create action for the default language
+    excludeRegex = regexToAlwaysExclude
+    if "excludeRegex" in entry:
+        excludeRegex += entry["excludeRegex"]
     action = baseAction | {
         "output": convertModuleToPath(entry["name"]) + "/src/main/res/values/localazy.xml",
         "includeKeys": list(map(lambda i: "REGEX:" + i, entry["includeRegex"])),
-        "excludeKeys": list(map(lambda i: "REGEX:" + i, regexToAlwaysExclude)),
+        "excludeKeys": list(map(lambda i: "REGEX:" + i, excludeRegex)),
```

Comment:
Thanks for the enhancement. You also have to update the action for the translations, [this line on develop](https://github.com/vector-im/element-x-android/blob/develop/tools/localazy/generateLocalazyConfig.py#L54):
From
```
            "excludeKeys": list(map(lambda i: "REGEX:" + i, regexToAlwaysExclude)),
```
to
```
            "excludeKeys": list(map(lambda i: "REGEX:" + i, excludeRegex)),
```

Reply:
Why do you compute again `excludeRegex`?

Reply:
🤦  I don't know why I thought they were 2 different functions. It should be fixed now.